docs/pgbench.md

```diff
@@ -0,0 +1,71 @@
+# pgbench
+
+[pgbench](https://www.postgresql.org/docs/10/pgbench.html) is a performance test kit for PostgreSQL.
+
+## Running pgbench
+
+Given that you followed instructions to deploy operator,
+you can modify [cr.yaml](../resources/crds/ripsaw_v1alpha1_pgbench_cr.yaml) to your needs.
```

Comment:
Throw the full name in here :

`s/cr.yaml/ripsaw_v1alpha1_pgbench_cr.yaml`

Reply:
This is actually consistent across other docs.

Reply:
yeah, I am systematically trying to get each updated...

Reply:
Not sure how much I care about it, but it _will_ be harder to cleanup later for PR #104

Reply:
It is mostly cosmetic.